src/migrations/dapps/1718991797670_trades-and-asset-trades-table.ts

```diff
@@ -28,9 +31,9 @@ export async function up(pgm: MigrationBuilder): Promise<void> {
       checks: { type: 'jsonb', notNull: true },
       signer: { type: 'varchar(42)', notNull: true },
       type: { type: TRADE_TYPE, notNull: true },
-      expires_at: { type: 'timestamp', notNull: true },
-      effective_since: { type: 'timestamp', notNull: true },
-      created_at: { type: 'timestamp', notNull: true, default: pgm.func('now()') }
+      expires_at: { type: 'timestamptz(3)', notNull: true },
```

Comment:
Now that we know the problem was the timezone, do we need to specify the precision as well?

Reply:
This gives us millisecond precision that will be the one used for all the communications with the api